roles/contiv_network/templates/netplugin.j2

```diff
@@ -1 +1 @@
-NETPLUGIN_ARGS='-plugin-mode {{netplugin_mode}} -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}}'
+NETPLUGIN_ARGS='-plugin-mode {{netplugin_mode}} -vlan-if {{netplugin_if}} -vtep-ip {{node_addr}} -fwd-mode {{fwd_mode}}'
```

Comment:
@abhinandanpb is this a recently added flag? 

Just want to make sure that the netplugin release installed (here: https://github.com/contiv/ansible/blob/master/roles/contiv_network/tasks/main.yml#L23) by current ansible supports this otherwise you will need to bump up the release number as well.